unifi/main.py

```diff
@@ -6,13 +6,14 @@
 
 import coloredlogs
 
-from unifi.cams import HikvisionCam, LorexCam, RTSPCam
+from unifi.cams import HikvisionCam, LorexCam, RTSPCam, ReolinkNVRCam
```

Comment:
If you install the dev dependencies, you can run isort to fix up the ordering

Reply:
@keshavdv I am really new to Python dev.  Mind telling me how to install the dev dependencies?